bankgreen/settings.py

```diff
@@ -44,7 +44,7 @@
 DEBUG = os.environ.get("DEBUG") == "True"
 
 ALLOWED_HOSTS = (
-    os.environ.get("ALLOWED_HOSTS").split(" ") if os.environ.get("ALLOWED_HOSTS") else []
+    os.environ.get("ALLOWED_HOSTS").split(" ")
```

Comment:
I'm not sure what benefit this brings. if `ALLOWED_HOSTS` isn't defined in the .env then it's going to throw an error now, since `.get()` will return `None`. If you'd like to phrase this more nicely, then you could use `.get("ALLOWED_HOSTS", "")` to return an empty string instead